test/common/api_utils.py

```diff
@@ -135,12 +143,8 @@ def api_node_select(client,
                 break
 
     # we now have a node ID list
-    if verbosity >= 3:
-        print "Node ID List:"
-        if node_id_list:
-            print node_id_list, '\n'
-        else:
-            print '**** Empty node ID list.\n'
+    if not  node_id_list:
```

Comment:
If we want to keep the logging the same, this would be:
{code}
if node_id_list:
    log.debug_7('Node ID List: Empty node ID List')
else:
    log.debug_7('Node ID List: %s', node_id_list)
{code}

(this may have been changed based on other input. If so, never mind!)